SwrveSDK/src/amazon/java/com/swrve/sdk/Swrve.java

```diff
@@ -0,0 +1,88 @@
+package com.swrve.sdk;
+
+import android.content.Context;
+
+import com.amazon.device.messaging.ADM;
+import com.swrve.sdk.config.SwrveConfig;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+/**
+ * Main implementation of the Amazon Swrve SDK.
+ */
+public class Swrve extends SwrveBase<ISwrve, SwrveConfig> implements ISwrve {
+    protected static final String SWRVE_AMAZON_TOKEN = "swrve.adm_token";
```

Comment:
For swrve.adm_token? Following the existing approach of swrve.gcm_token. I believe this is expected server side in the user properties.